code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -172,7 +172,7 @@
 			if(attacker_style && attacker_style.disarm_act(H, src))
 				return 1
 			else
-				add_logs(M, src, "disarmed")
+				add_logs(M, src, "disarmed", admin = 0)

```

Comment:
We don't really need to see it when it happens, it's enough to have it logged in the mob attacklogs so we can see it if it becomes relevant.